src/registry.js

```diff
@@ -0,0 +1,296 @@
+"use strict";
```

Comment:
`getDefinition` and `undefine` are extensions. The rest of the changes were to make it compatible with domino instead of the actual DOM.